### ISSUE TITLE
PHP 8 support

### ISSUE THREAD

**Issue by mathewrapid** [advisors-excel-llc/salesforce-rest-sdk#84]
Composer gives this error
```
- ae/salesforce-rest-sdk[2.0.0, v2.0.1] require php ^7.2 -> your php version (8.0.2) does not satisfy that requirement.
```

**Comment by hanishsingla** [comment]
@bluedot74 @curiosity26 @NickALafrance Is this bundle being managed? I would like to push PRs to make it PHP8 compatible if needed

**Comment by bluedot74** [comment]
> @bluedot74 @curiosity26 @NickALafrance Is this bundle being managed? I would like to push PRs to make it PHP8 compatible if needed

No, I don't think so. I'd recommend forking it if you wanna keep it going.

**Comment by hanishsingla** [comment]
For anyone looking for further support for PHP8, you can check https://packagist.org/packages/codegenus/salesforce-rest-sdk
